packages/@aws-cdk/aws-codepipeline-actions/lib/cloudformation/pipeline-actions.ts

```diff
@@ -513,6 +513,487 @@ export class CloudFormationDeleteStackAction extends CloudFormationDeployAction
   }
 }
 
+/**
+ * Determines how IAM roles are created and managed
+ */
+export enum StackSetPermissionModel {
+  /**
+   * You must create administrator and execution roles to deploy to target accounts
+   */
+  SELF_MANAGED = 'SELF_MANAGED',
+
+  /**
+   * AWS CloudFormation StackSets automatically creates the IAM roles required to deploy to accounts managed by AWS Organizations. This requires an
+   * account to be a member of an Organization
```

Comment:
Please brake comments on sentence boundaries:

```suggestion
   * AWS CloudFormation StackSets automatically creates the IAM roles required to deploy to accounts managed by AWS Organizations.
   * This requires an account to be a member of an Organization.
```

Same comment everywhere below.